index.md

```diff
@@ -11,13 +11,13 @@ forecasting the weather, to predicting the flow field around aircraft,
 through to predictive simulations for biomedical applications, amongst
 many other examples.
 
-PDESoft 2024 is an opportunity for users and developers of software
+PDESoft 2024 conference is an opportunity for users and developers of software
```

Comment:
```suggestion
PDESoft 2024 is an opportunity for users and developers of software
```
I think this was fine before. Most people will know that "PDESoft 2024" is a conference, and if you add the word "conference" you will need to say something like "The PDESoft 2024 conference" (adding definite article)...